README.md

```diff
@@ -75,10 +76,4 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 ## Contact
 
-### Intercom
-
-If you have further questions, want to provide feedback or you are running into issues, log in to the [App Center](https://appcenter.ms) portal and use the blue Intercom button on the bottom right to start a conversation with us.
-
-### Twitter
-
-We're on Twitter as [@appcenter](https://www.twitter.com/vsappcenter).
+We're on Twitter as [@vsappcenter](https://www.twitter.com/vsappcenter). Additionally you can reach out to us on the [App Center](https://appcenter.ms) portal by using the blue Intercom button on the bottom right to start a conversation.
```

Comment:
Maybe point to `https://appcenter.ms/apps` instead to make sure people get directly to the login page in case they are not logged in right now?

Reply:
(A bit out of scope of this PR though...)

Reply:
@janpio yep, sorry for that, gave me reason to improve a few things 😳